### PR TITLE
Doc - Remove standalone cluster doc

### PR DESCRIPTION
**Pull request by kcoriordan**
Signed-off-by: kcoriordan <koriordan@vmware.com>

## What this PR does / why we need it
-Removes standalone cluster topics from table of contents
-Removes references in topics to standalone cluster
-Adds a note (using an include) to the following topics to indicate that they relate to managed clusters and provide a link to the unmanaged cluster doc which will be merged in #2820 
       - Support matrix
       - Getting Started (Management Cluster)
       - Planning your installation
       - Installing TCE
       
-Note the standalone cluster topics have not been deleted and are still available through search or saved bookmarks
Related to: #2266, #2820

## Details for the Release Notes (PLEASE PROVIDE)

```none

```

## Which issue(s) this PR fixes

Related to: #2266

### CODE REVIEW

**Review by jpmcb — COMMENTED** [review]
Not sure why the Netlify site didn't build, but pulled this down, built it with
```
cd docs/site && hugo build
```
and looking good to me. Will let others comment as well

---

Question for @garrying regarding the Algolia search indexing; does the search dynamically update itself based on what's in the site? Will the standalone-cluster pages still show up in the search if we remove it from the TOC?

**Comment by garrying** [comment]
@jpmcb Search index will dynamically update based on what's in the site (on next crawl). I'm looking into adding a synonym of `standalone -> unmanaged` to help with surfacing content in Algolia's index.

**Comment by joshrosso** [comment]
> Not sure why the Netlify site didn't build, but pulled this down, built it with
> 
> ```
> cd docs/site && hugo build
> ```
> 
> and looking good to me. Will let others comment as well
> 
> Question for @garrying regarding the Algolia search indexing; does the search dynamically update itself based on what's in the site? Will the standalone-cluster pages still show up in the search if we remove it from the TOC?

```

**Comment by joshrosso** [comment]
> Not sure why the Netlify site didn't build, but pulled this down, built it with
> 
> ```
> cd docs/site && hugo build
> ```
> 
> and looking good to me. Will let others comment as well
> 
> Question for @garrying regarding the Algolia search indexing; does the search dynamically update itself based on what's in the site? Will the standalone-cluster pages still show up in the search if we remove it from the TOC?

```
Failed during stage 'checking build content for changes': Canceled build due to no content change
```

seems Netlify believes there has been no content change, which is surprising. Will kick off tests again.

File: docs/site/content/docs/latest/docker-monitoring-stack.md [hunk 1]
```diff
@@ -6,37 +6,19 @@ From a dependency perspective, Prometheus and Grafana need an Ingress, or a HTTP
...
 - Create a namespace for MetalLB
 - Deploy the components of MetalLB
 - Create a ConfigMap which contains the list of addresses to use for VIPs / Load Balancer IP Addresses.
 
-Deployment of the Tanzu Community Edition standalone cluster and metallb are beyond the scope of this document.
+Deployment of the Tanzu Community Edition cluster and metallb are beyond the scope of this document.
 
-It is also recommended that reader familiarise themselves with the [working with packages](/docs/latest/package-management.md) documention as we will be using packages extensively in this procedure.
-
-## Examining the Tanzu Community Edition Standalone environment
```
[the remark this answers is not in the document]
Comment (kcoriordan) [reply docs/site/content/docs/latest/docker-monitoring-stack.md:19]: As this guide is already very long and to reduce repetition across the docs, instead I added in a link to the Deploying workload clusters topic - this should have everything needed.

**Comment by kcoriordan** [comment]
preview here: https://deploy-preview-2862--quirky-franklin-8969be.netlify.app/docs/latest/docker-monitoring-stack/

not sure why the Netlify link doesn't show in checks, this happens sometimes and I manually add the PR# to the Netlify preview URL

**Review by jpmcb — DISMISSED** [review]
This looks good to me! Do we want to pull the trigger on getting these docs out? My vote is yes, sooner the better.

**Comment by kcoriordan** [comment]
@jpmcb fixed merge conflict

**Comment by jpmcb** [comment]
@joshrosso - thoughts on getting this out now? Or do we want to wait on this after v0.10.0? I think it'd be fine to go out now

**Comment by qnetter** [comment]
When do we currently plan on starting to version the docs?

**Comment by joshrosso** [comment]
> When do we currently plan on starting to version the docs?

https://github.com/vmware-tanzu/community-edition/issues/882

**Review by jpmcb — DISMISSED** [review]

**Review by davidvonthenen — APPROVED** [review]
Resolved merge conflict with this commit:
https://github.com/vmware-tanzu/community-edition/pull/2862/commits/b983d0744632ba825c75c2a0602f69de328840ab

**Review by stmcginnis — APPROVED** [review]
Looks like everything is covered.

**Pull request merged**